google-cloud-container/README.md

```diff
@@ -113,6 +113,6 @@ Apache 2.0 - See [LICENSE] for more information.
 [TESTING]: https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/TESTING.md#testing-code-that-uses-container
 
 [cloud-container]: https://cloud.google.com/container/
```

Comment:
Since this URL redirects, I would suggest using what it redirects to, https://cloud.google.com/kubernetes-engine/

Reply:
Done.